server/apps/distributed-app/docs/modules/ROOT/pages/operate/webadmin.adoc

```diff
@@ -551,6 +551,78 @@ Response codes:
 * 400: The user is invalid
 * 404: The user is unknown
 
+== Administrating vacation settings
+
+=== Get vacation settings
+
+....
+curl -XGET http://ip:port/vacation/usernameToBeUsed
+....
+
+Resource name usernameToBeUsed representing valid users, hence it should
+match the criteria at xref:confgure/usersrepository.adoc[User Repositories documentation]
+
+The response will look like this:
+
+....
+{
+  "enabled": true,
+  "fromDate": "2021-09-20T10:00:00Z",
+  "toDate": "2021-09-27T18:00:00Z",
+  "subject": "Out of office",
+  "textBody": "I am on vacation, will be back soon.",
+  "htmlBody": "<p>I am on vacation, will be back soon.</p>"
+}
+....
+
+Response codes:
+
+* 200: The vacation settings were successfully retrieved
+* 400: The user name is invalid
+
+=== Update vacation settings
+
+....
+curl -XPOST http://ip:port/vacation/usernameToBeUsed
+....
+
+Request body must be a JSON structure as described above.
+
+If any field is not set in the request, the corresponding field in the existing vacation message is left unchanged.
+
+Response codes:
+
+* 204: The vacation settings were successfully updated
+* 400: The user name or the payload is invalid
+
+=== Patch vacation settings
+
+....
+curl -XPATCH http://ip:port/vacation/usernameToBeUsed
```

Comment:
> -> null reset to null
-> absent ignores.

I Would prefer it this way, but unfortunately the JSON deserializer cannot distinguish these cases. It results in null or Optional.empty regardless, whether I specify the property or omit it. This is the reason why I added two methods to handle null/absent properties, one to update (POST) and one to reset (PATCH).

But overall, I believe the update (POST) is the more useful method, and there is a workaround to DELETE and then POST a partial object again if someone actually needs the reset functionality. So I will drop PATCH, since having only one method is probably less confusing to the end user.